src/@providers/gcp/modules/deployment.ts

```diff
@@ -40,12 +40,13 @@ export class GoogleCloudDeploymentModule extends ResourceModule<
     this.deployments = [];
     const labels: Record<string, string> = {};
 
-    for (const service of this.inputs?.services || []) {
+    for (const [index, service] of (this.inputs?.services || []).entries()) {
       const deployment_name = (this.inputs?.namespace || 'ns') + '-' +
-        (this.inputs?.name.replaceAll('/', '-') || 'deleting') + `-${service.port || 80}`;
-      const deployment = new CloudRunV2Service(this, `${deployment_name}-deployment`, {
+        (this.inputs?.name.replaceAll('/', '-') || 'deleting');
+      const resource_name = `${deployment_name}-svc-${index}`;
```

Comment:
I used `service.port` before because we need to also construct this name in the `service` module where we don't actually know which index it would be in the list - so if we update this to 

```
const resource_name = `${deployment_name}-svc-${service.port || 80}`;
```

we could change https://github.com/architect-team/arcctl/pull/89/files#diff-d16a7edc1c8f99048a9c7e26b6786900571e2e7165997e82c80bf1eb2afdfd64R22
to match